addons/account/models/account_reconcile_model.py

```diff
@@ -199,11 +195,11 @@ def _get_write_off_move_lines_dict(self, st_line, move_lines=None):
         }
         new_aml_dicts.append(writeoff_line)
 
-        if self.tax_id:
-            writeoff_line['tax_ids'] = [(6, None, [self.tax_id.id])]
-            tax = self.tax_id
+        if self.tax_ids:
+            writeoff_line['tax_ids'] = [(6, None, self.tax_ids.ids)]
+            tax = self.tax_ids
             if self.force_tax_included:
-                tax = tax.with_context(force_price_include=True)
+                tax = tax[0].with_context(force_price_include=True)
```

Comment:
actually, behavior with multiples taxes and force_price_include result in incorrect number. So we decided that we could not check the force_tax_included if there was more than one tax. This is reflected in the 
```
def _onchange_tax_ids(self):
        self.show_force_tax_included = False if len(self.tax_ids) != 1 else True
        if len(self.tax_ids) != 1:
            self.force_tax_included = False
```
So we can assume that if we have self.tax_ids and self.force_tax_include than we should only have one tax (and if there is more, we should not take all of them), so we can directly take the tax with tax[0] instead of doing a browse again

What do you think?

Reply:
OK... then please add those comments within the code (both in the onchange and here) to make things explicit ;-)